src/System/Hapistrano/Core.hs

```diff
@@ -110,7 +110,7 @@ getProgAndArgs cmd = do
       Nothing ->
         (renderShell configShellOptions, ["-c", cmd])
       Just SshOptions {..} ->
-        ("ssh", [sshHost, "-p", show sshPort, cmd])
+        ("ssh", sshArgs ++ [sshHost, "-p", show sshPort, cmd])
```

Comment:
should `sshArgs` overwrite the `port` option too?

Reply:
@juanpaucar Good catch! We're lucky :four_leaf_clover: SSH won't complain if it contains a flag duplicated. Also, it seems that it will take the latter rather than the former. Therefore, `sshArgs` won't overwrite `port`. 

I tried: `ssh -p 2222 -p 22 user@127.0.0.1` and it tried to access via `22` rather than `2222`.